Tabloid/client/src/modules/reactionManager.js

```diff
@@ -0,0 +1,12 @@
+const reactionUrl = "/api/reaction";
+
+
+export const addReaction = (reaction) => {
+    return fetch(reactionUrl, {
```

Comment:
if the controller requires auth, you're going to need an auth token to pass to the fetch call to do this properly.